### DISCUSSION TITLE
How can I set reserve storage percentage of nodes in longhorn at the time of deployment via helm chart

### DISCUSSION THREAD

**Discussion by ravisingh-adeptia** [longhorn/longhorn#5957] [Q&A]
How can I set reserve storage percentage of nodes in longhorn at the time of deployment via helm chart.
Currently I am deploying longhorn via helm chart by default it is acquiring 30% of storage as reserve storage, and  on node no  other application is deployed so I am suggesting that reserve storage amount is of larger size.
So I want to know if there is any way we can reduce reserve storage amount at the time of deployment.

**Comment by ChanYiLin** [comment]
Currently this setting can be only changed on the UI after longhorn is deployed

**Comment by ChanYiLin** [comment]
I have created a issue to add this setting to helm chart
https://github.com/longhorn/longhorn/issues/5958
thanks!

**Comment by ChanYiLin** [comment]
Hi @ravisingh-adeptia 
I just found maybe you can try to add the setting into the helm chart yourself
The default setting is set in this template
- https://github.com/longhorn/longhorn/blob/master/chart/templates/default-setting.yaml
And the value comes from 
- https://github.com/longhorn/longhorn/blob/master/chart/values.yaml#L107
You can update the helm chart to add the setting and give it the value

The field name for the setting is `storage-reserved-percentage-for-default-disk`
You can add 
`storage-reserved-percentage-for-default-disk: 30` to the default-setting.yaml 

I think this workaround could work

**Reply by ravisingh-adeptia** [reply]
Hi @ChanYiLin 
I have tried the above proposed solution.
I have added storage-reserved-percentage-for-default-disk: 30 to the default-setting.yaml but it didn't worked.

Also on listing settings via command kubectl get settings.longhorn.io -n longhorn-system
all other settings are getting listed but setting for reserve storage doesn't shows up

**Reply by ChanYiLin** [reply]
Hi @ravisingh-adeptia 
Sorry, I just found this setting is in v1.5.0 which will be released in June
For the previous version, we don't have this setting.
cc @innobead
